src/garage/tf/algos/npo.py

```diff
@@ -68,6 +67,9 @@ class NPO(BatchPolopt):
         flatten_input (bool): Whether to flatten input along the observation
             dimension. If True, for example, an observation with shape (2, 4)
             will be flattened to 8.
+        fit_baseline_order (str): Whether to fit baseline before or after,
```

Comment:
this needs much better documentation. it assumes the user knows all about the internals of NPO.

Please make the point of reference the optimization step, not the prediction. In the normal case, we fit the baseline after optimizing the policy. In the strange case, we fit the baseline after optimizing the policy.